apidef/oas/security.go

```diff
@@ -294,6 +294,22 @@ func (s *OAS) extractCustomPluginTo(api *apidef.APIDefinition) {
 	api.AuthConfigs[apidef.CoprocessType] = authConfig
 }
 
+func (s *OAS) fillGoPlugin(api apidef.APIDefinition) {
+	goPlugin := &GoPlugin{}
+	goPlugin.Enabled = api.UseGoPluginAuth
```

Comment:
Could we just use this value in order to avoid the allocation if ShouldOmit(goPlugin)?